json_test.go

```diff
@@ -78,9 +79,10 @@ func TestSortJSON(t *testing.T) {
 }
 
 func testCompactJSON(t *testing.T, input, want string) {
-	got := string(CompactJSON([]byte(input), nil))
+	bytes := CompactJSON([]byte(input), nil)
+	got := string(bytes)
 	if got != want {
-		t.Errorf("CompactJSON(%q):\n want: %q\n got: %q", input, want, got)
+		t.Errorf("CompactJSON(%q):\n want: %q\n got: %q\n bytes: % X", input, want, got, bytes)
```

Comment:
```suggestion
		t.Errorf("CompactJSON(%q):\n want: %q\n got: %q\n bytes: %x", input, want, got, bytes)
```
I assume? (with the recent lower-/uppercase thing?)

Reply:
From https://pkg.go.dev/fmt
```
' '	(space) leave a space for elided sign in numbers (% d);
	put spaces between bytes printing strings or slices in hex (% x, % X)
```

So this is fine.